src/controllers/tripProfileController.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/prefer-default-export */
+import * as ApplicationError from '../utils/errors/applicationsErrors';
+import {
+  createdResponse,
+  notFoundResponse,
+  successResponse
+} from '../utils/responseHandler';
+import {
+  fetchTripProfile,
+  findOneTripInfo,
+  isSavingTripInfo
+} from '../services/tripProfileInfo';
+
+export const addProfileInfo = async (trip, req, res) => {
+  const tripProfile = await isSavingTripInfo(trip, req);
```

Comment:
You should do this in trip.controller.js file

Reply:
Why? this function belongs to tripProfileInfo, not the trip itself.  Besides, to keep putting many codes in the trip controller is not ideal as it's getting very big